### PR TITLE
[Doc] Add inference speed results

### PR DESCRIPTION
**Pull request by liqikai9**
## Motivation

Add docs related to inference speed results.

## Modification

Modify `README.md`, `README_CN.md` and add `docs/inference_speed_results.md`.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -164,6 +164,98 @@ https://user-images.githubusercontent.com/15977946/124654387-0fd3c500-ded1-11eb-
...
 更多详情可见 [基准测试](docs/benchmark.md)。
 
+## 推理速度总结
+
+这里总结了 MMPose 中主要模型的复杂度信息和推理速度，包括模型的计算复杂度、参数数量，以及以不同的批处理大小在 CPU 和 GPU 上的推理速度。
+
+<details open>
+<summary>(点击收起)</summary>
+
+| 算法 | 模型 | 配置文件 | 输入分辨率 | 全类别平均正确率 | 浮点数运算次数<br>(10亿) | 参数数量<br>(百万) | GPU 上的推理速度<br>(每秒处理的帧数)| GPU 上的推理速度<br>(每秒处理的帧数, 批处理大小为10) | CPU 上的推理速度<br>(每秒处理的帧数) | CPU 上的推理速度<br>(每秒处理的帧数, 批处理大小为10) |
```
Comment (ly015) [inline README_CN.md:174]: Using English for the title row is okay here.

File: README.md [hunk 2]
```diff
@@ -165,6 +165,98 @@ We demonstrate the superiority of our MMPose framework in terms of speed and acc
 
 More details about the benchmark are available on [benchmark.md](docs/benchmark.md).
 
+## Inference speed summary
+
+We show the summary of complexity information and inference speed for the major models in MMPose, including FLOPs, parameter counts and inference speeds on both CPU and GPU devices with different batch sizes.
+
+<details open>
+<summary>(click to collapse)</summary>
```
Comment (ly015) [inline README.md:173]: The table is too massive to show in the readme. I suggest only keeping a short description and the link to the markdown file here.

File: README_CN.md [hunk 3]
```diff
@@ -164,6 +164,98 @@ https://user-images.githubusercontent.com/15977946/124654387-0fd3c500-ded1-11eb-
 
 更多详情可见 [基准测试](docs/benchmark.md)。
 
+## 推理速度总结
```
Comment (ly015) [inline README_CN.md:167]: Only keep description and link here.

File: README.md (hunk 2 above)
Comment (liqikai9) [reply README.md:173]: Good suggestion. I will change it.

File: README_CN.md (hunk 3 above)
Comment (liqikai9) [reply README_CN.md:167]: OK, I will change it.

**Review by ly015 — APPROVED** [review]

**Pull request merged**